src/amo/css/OverallRating.scss

```diff
@@ -6,22 +6,28 @@
 
 $star-size: 46px;
 
-.OverallRating-choice {
+.OverallRating-choices button {
   background: url('../img/ratings/big-star.svg') center/$star-size no-repeat;
   cursor: pointer;
   height: $star-size + 10px;
   padding-top: 80px;
   width: $star-size + 10px;
 }
 
+.OverallRating-choices button:disabled {
```

Comment:
The dilemma with always setting this class is that there are two types of buttons: 1) ones that you can click on and 2) ones that are informative but that you can't click on. This seems like the easiest way to support both cases. Each case is pretty different visually, mostly in all the hover logic.